.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "3.9"
```

Comment:
I am guessing this is related to `numpy`. Depending on final design the change might be needed here for tests and not in `setup.cfg` for downstream package consumers.

Reply:
It's because of numpy's version that is not compatible with `3.7`. I thought bumping to `3.9` was reasonable as it is the default on most platforms as of now.

I'll update the `setup.cfg`.

Reply:
Sounds good.